### PR TITLE
[IOPAE-1181] IP inputs unclickable

### PR DESCRIPTION
**Pull request by carlo-bussi**
#### List of Changes

Now the IP inputs are unclickable outiside the edit mode

#### Motivation and Context

QA

#### How Has This Been Tested?

Local env

#### Screenshots (if appropriate):

https://github.com/pagopa/io-services-cms/assets/157382565/5232c160-4b80-4eba-906a-be0802396e1a

#### Checklist:

- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.

### CODE REVIEW

File: apps/backoffice/src/components/forms/controllers/text-field-array-controller.tsx [hunk 1]
```diff
@@ -119,6 +122,13 @@ export function TextFieldArrayController({
                     {...register(getFieldName(index))}
                     {...props}
                     margin="normal"
+                    sx={
```
Comment (gunzip) [inline apps/backoffice/src/components/forms/controllers/text-field-array-controller.tsx:125]: what about adding a `disabled` attribute?

Comment (carlo-bussi) [reply apps/backoffice/src/components/forms/controllers/text-field-array-controller.tsx:125]: we want to keep the standard input style. add disable and change its style would be more complicated

**Review by giuseppedipinto — COMMENTED** [review]
Please check this component in service create/update mode: in both cases we have a regression in the behaviour because we are missing out the edit functionality as there is no possibility to add or edit a new or existent IP field.

**Review by giuseppedipinto — CHANGES_REQUESTED** [review]
It still doesn't work, steps to reproduce the problem:

1. Try creating a service
2. add an ip/cidr field
3. add a second ip/cidr field
4. click _Cancel_ button
5. add an ip/cidr field
6.❌ Fail edit

- Same behaviour in service edit use case.
- Same behaviour _(maybe even worse)_ in Api Key editing.

**Review by giuseppedipinto — APPROVED** [review]
LGTM

**Pull request merged**